src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -124,48 +122,101 @@ public static void tearDown() {
      */
     @Test
     public void canSimulateApiUserFlow() {
-        // create otp user as api user
+
+        // obtain bearer token
+        String endpoint = String.format("api/secure/application/%s/authenticate?username=%s&password=%s",
+            apiUser.id,
+            apiUser.email,
+            TEMP_AUTH0_USER_PASSWORD);
+        HttpResponse<String> getTokenResponse = mockAuthenticatedPost(endpoint,
+            apiUser,
+            ""
+        );
+        System.out.println(getTokenResponse.body());
```

Comment:
We should avoid `System.out` and use `LOG.info`.

Reply:
Updated.